frontend/shared/src/utils/string.utils.ts

```diff
@@ -26,3 +26,16 @@ export const getBooleanValueFromString = (value?: string): boolean | null => {
 
 export const isEmpty = (value?: string): boolean =>
   typeof value === 'string' ? value?.trim().length === 0 : Boolean(!value);
+
+export const getNumberValue = (s: string): number =>
+  Number(s?.toString().replace(/,/, '.'));
```

Comment:
Since `s` is always a string, conditional checking is not required.
```suggestion
  Number(s.toString().replace(/,/, '.'));
```